### PR TITLE
New Data .py

### PR DESCRIPTION
**Pull request by JPOJ69**
New python file was added to the gitlab.

### CODE REVIEW

**Comment by JPOJ69** [comment]
looks good to me

**Review by JPOJ69 — COMMENTED** [review]
missing some code you should probably retry

File: data_collection.py [hunk 1]
```diff
@@ -0,0 +1 @@
+requests
```
Comment (JPOJ69) [inline data_collection.py:1]: needs work

**Review by Yemi99 — COMMENTED** [review]
Nice.

File: data_collection.py (hunk 1 above)
Comment (JPOJ69) [reply data_collection.py:1]: a lot
of work

**Pull request merged**